### PR TITLE
[MINOR] Fix config type of imagePullSecrets in values.yaml for helm

### PR DESCRIPTION
**Pull request by dev-lpq**
### What changes were proposed in this pull request?

Fix config type of `imagePullSecrets` in `values.yaml` for helm.

### Why are the changes needed?

The `imagePullSecrets` of `values.yaml` is wrong.

### Does this PR introduce _any_ user-facing change?

No any user-facing change.

### How was this patch tested?

No tests.

### CODE REVIEW

File: charts/celeborn/values.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ image:
   pullPolicy: Always
   tag: 0.1.1-6badd20
 
-imagePullSecrets: {}
+imagePullSecrets: []
```
Comment (SteNicholas) [inline charts/celeborn/values.yaml:29]: Could you also fix `imagePullSecrets` in`tests/kubernetes-it/docker/helm/values.yaml`?

Comment (dev-lpq) [reply charts/celeborn/values.yaml:29]: Sure, updated

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @dev-lpq update.

**Review by FMX — APPROVED** [review]
LGTM. Merged into main(v0.5.0) and branch-0.4(v0.4.0).

**Pull request closed without merging**